### ISSUE TITLE
Install?  Quickstart?

### ISSUE THREAD

**Issue by BevanR** [Adezandee/cucumber-mink#163]
How can I use it?  How do I get started?  Is there an install process?  Is there a boilerplate or quickstart example repo?  I couldn't find any documentation about this.

**Comment by Adezandee** [comment]
Hi @BevanR, you can find documentation and quickstart here http://cucumber-mink.js.org/.
Example implementation inside the repo here https://github.com/Adezandee/cucumber-mink/tree/master/test/features

**Comment by fullstackwebdev** [comment]
Hi I am unable to start either.  It seems to be ignoring my support files specifically mink.js

The documents don't describe how to include the mink.js, but the older README does say, " --require ./features/support/mink.js ..." -- however that doesn't work. The new documentation doesn't appear to say. I also tried Inject.

Did I miss this somewhere?

**Comment by Adezandee** [comment]
`cucumber` cli with `npm i -g cucumber` will autoload support code automatically (at least last time I checked). Otherwise, if using support code with `cucumber-mink` you should disable inject and pass in `--require mink.js`like this:
``` sh
cucumber-mink --inject=false -- -r mink.js
```

**Comment by davidje** [comment]
Thanks, can I ask another question; how do I use protractor.cucumber.js?

**Comment by davidje** [comment]
I was able to get cucumber-mink to load my step_definitions, but not cucumber js.  Isn't cucumber.js suppose to see my ./features/support/mink.js?

This works:
```
$ ./node_modules/.bin/cucumber-mink --inject=false -- -r ./features/support/mink.js -r ./features/step_definitions/test.js
```

This doesn't work 

```
$ ./node_modules/.bin/cucumberjs
Warnings:

1) Scenario: Invalid Login - features/campaigns/10-login.feature:9
   Step: Given I browse "http://localhost:9000" - features/campaigns/10-login.feature:6
   Message:
     Undefined. Implement with the following snippet:

       this.Given(/^I browse "([^"]*)"$/, function (arg1, callback) {
         // Write code here that turns the phrase above into concrete actions
         callback(null, 'pending');
       });

. . . 32 different undefined 

. . . 
3 scenarios (3 undefined)
32 steps (32 undefined)
0m00.000s


```

**Comment by Adezandee** [comment]
@davidje, indeed. the cucumber cli should autoload support files. see https://github.com/cucumber/cucumber-js#requiring-support-files

**Comment by BevanR** [comment]
I got this a step further, but ran into what appears to be a compatibility issue; #169

**Comment by BevanR** [comment]
The trick to getting it working is to explicitly require support files.  E.g. `cucumberjs -r support/mink.js features/`
